### DISCUSSION TITLE
Loading problem with Solar System Editor

### DISCUSSION THREAD

**Discussion by stephanpls** [Stellarium/stellarium#2143] [Q&A]
In the Solar System Editor, tab Configuration file, I encounter these problems:

1. If I try the 'Import and replace the Solar System minor objects to file..' function, I see that the destination file (...Users\AppData\Roaming\Stellarium\data\ssystem_minor.ini) indeed only contains the data I wanted to add. However, Stellarium ignores this data so it is not found with Search (F3) nor in the Location window (F6).

2. If I try the 'Import and add Solar System minor objects from file..' function, I see in the destination file that the data I wanted to add is not added at all.

However, if I add my data manually to the destination file (with Notepad) Stellarium reads it and is working with it as expected.

**Comment by alex-w** [comment]
Please share log.txt file

**Comment by stephanpls** [comment]
Here it is. Meanwhile, maybe a solution is adding a crlf after every record I add to my ssystem_minor.ini (I fill it with my own software). Could that be the cause?

[log.txt](https://github.com/Stellarium/stellarium/files/7799029/log.txt)
.

**Reply by alex-w** [reply]
Thanks! Please try import the data and share log after usage of SSE plugin

**Reply by gzotti** [reply]
yes, I think (without digging in the code) the lines must be separated by CRLF, the .ini is a PC format. And we use an empty line between records, but this should technically not matter.

**Comment by stephanpls** [comment]
Here it is, a very fresh one. I tried my presumed solution, so I filled my ssystem_minor.ini with crlf at every record end, but it seems that it does not help.
[log.txt](https://github.com/Stellarium/stellarium/files/7799066/log.txt)
.

**Reply by gzotti** [reply]
Did you use the SSE plugin during this run?

**Comment by stephanpls** [comment]
Yes

**Reply by alex-w** [reply]
Hmm… Would you share your data file for experiments?

**Comment by stephanpls** [comment]
Here you go (zipped because .ini is not accepted here). As said in another thread, I'm working on data for spacecraft, now including also  planet flybys. Very nice views, you see rotating planets with its moons approaching.....  I'll share all the data later if it works for normal users, i.e. if we get SSE working with my .ini.

[ssystem_spacecraft_and_flybys.zip](https://github.com/Stellarium/stellarium/files/7799096/ssystem_spacecraft_and_flybys.zip)

**Reply by alex-w** [reply]
I can confirm the issue at stage of loading the data from file on macOS

**Reply by gzotti** [reply]
Convert to UTF-8 and it works.

**Reply by alex-w** [reply]
Well, we should check the encoding of opening files and add warning when encoding is unexpected.
